docs/Makefile

```diff
@@ -0,0 +1,19 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
We already have a Make file for this in [the repo](https://github.com/DesignSafe-CI/portal/blob/master/.docs/Makefile)